wiki/Tournaments/MWC/2023_4K/en.md

```diff
@@ -0,0 +1,200 @@
+
+### Match instructions
+
+1. A referee will create a multiplayer room 10 minutes before the scheduled match time. Teams must join the lobby in that period.
+   - The room settings are `Game mode: "osu!mania"`, `Team mode: "Team Vs"` and `Score mode: "ScoreV2"`. The room name must follow the pattern of `MWC4K2023: ({Red Team}) VS ({Blue Team})`. The team mentioned first in the room name must be the red team, the team mentioned second in the room name must be the blue team.
+2. Each captain will first protect one beatmap from the pool. Captains then proceed to each ban one beatmap from the pool, and they may not ban any of the maps that were protected. The banned beatmaps may not be picked by any team for the entire duration of the match.
+3. Each captain must use `!roll` once in the match's chatroom.
+   - The winner of the `!roll` starts protecting one beatmap, followed by the loser of the `!roll` to protect a beatmap. They may not protect the same beatmap. Protected beatmaps may not be banned.
+   - The winner of the `!roll` starts picking the first beatmap of the match.
+   - The loser of the `!roll` starts banning one beatmap, followed by the winner of the `!roll` to ban a beatmap.
+4. After bans are decided, both teams will take turns in picking a beatmap from the mappool.
+5. Teams will be allowed 2 minutes to pick a beatmap and 2 minutes to press the `Ready` button on their client. If a team takes more time than alloted for either action, the procedures adopted will be as follows:
+   - For the first occurrence:
+     - The team will receive a verbal warning from the referee; in addition, the referee may opt to forcefully use the team's tactical timeout to allow for the team to pick.
+   - On subsequent occurrences:
+     - For a pick timer: a random map will be chosen from the mappool using `!roll X`, where X is the number of beatmaps that were neither picked nor banned, excluding the tiebreaker.
+     - For a ready timer: the referee will issue the `!mp start 10` command, regardless of how many players from each team are present in the lobby, using `!mp kick` on any extra players for each team, starting from the top (i.e. the first valid player combination for each team will be forced to play the pick). The results for such games are to be taken as is.
+     - Repeat offenders may receive further sanctions from the tournament managers.
+6. In the case of a team starting a map with less than the optimal number of players on their roster due to an unforeseen circumstance (e.g. a technical issue or unavoidable player absence), the team in question will be granted the right to fill their roster before play commences. The remainder of the original roster must stay the same, i.e. only player additions are allowed.
+7. Each team will receive one "tactical timeout" of 2 minutes, to be used as extra time to pick, ban, or ready for a beatmap. *The tactical timeout is optional.*
```

Comment:
it's intended that protections are excluded, it should probably be clarified explicitly

```suggestion
7. Each team will receive one "tactical timeout" of 2 minutes, to be used as extra time to protect, pick, ban, or ready for a beatmap. *The tactical timeout is optional.*
```